code/game/jobs/job_exp.dm

```diff
@@ -6,7 +6,7 @@ GLOBAL_LIST_INIT(role_playtime_requirements, list(
 	ROLE_POSIBRAIN = 5, // Same as cyborg job.
 	ROLE_SENTIENT = 5,
 	ROLE_ERT = 40, // High, because they're team-based, and we want ERT to be robust
-	ROLE_DEATHSQUAD = 10,
+	ROLE_DEATHSQUAD = 50, // Higher, see ERT and also they're OP as heck
 	ROLE_TRADER = 20, // Very high, because they're an admin-spawned event with powerful items
```

Comment:
That ones my bad, meant to add it when I changed it. Since Deathsquads can be spawned now randomly from ghosts now if the admin chooses thought they should be at least the same or higher than ERT